-inputs.tf

```diff
@@ -35,11 +41,17 @@ variable "required_tags_enabled" {
 
 variable "required_tags" {
   description = "Map of tag keys, and optionally values, that are required."
-  type        = map
+  type        = map(any)
   default     = {}
 }
 
 variable "source_recorder" {
   description = "The AWS Config Recorder for this region. This is used to set the correct application order/dependency chain."
   type        = string
+}
+
+variable "daysToExpiration" {
+  description = "Defines the number of days before expiration before Config flags a rule as noncompliant."
```

Comment:
should this be "flagging a rule as noncompliant" or "a certificate as noncompliant" ?